model/k8s/resources.yaml

```diff
@@ -74,6 +74,9 @@ groups:
     attributes:
       - ref: k8s.deployment.uid
       - ref: k8s.deployment.name
+      - ref: k8s.deployment.label
```

Comment:
I know that's not the case so far but I wonder if we should consider having labels as `opt-in`.

@open-telemetry/semconv-k8s-approvers @open-telemetry/specs-semconv-approvers thoughts?

Reply:
I'm in favor of making `labels` opt-in (assuming we are discussing this change for all k8s resource types). They can be numerous and sometimes mutable, which may introduce high cardinality. Marking them as opt-in like annotations seems appropriate and consistent to me.